src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetDayName.cs

```diff
@@ -47,7 +47,8 @@ public void GetDayName(DateTimeFormatInfo format, string[] expected)
 
         public void GetDayName_Invalid()
         {
-            Assert.Throws<ArgumentOutOfRangeException>(() => new DateTimeFormatInfo().GetDayName((DayOfWeek)(-1))); // DayOfWeek is invalid
+            Assert.Throws<ArgumentOutOfRangeException>("value", () => new DateTimeFormatInfo().GetDayName(DayOfWeek.Sunday - 1)); // DayOfWeek is invalid
+            Assert.Throws<ArgumentOutOfRangeException>("value", () => new DateTimeFormatInfo().GetDayName(DayOfWeek.Saturday + 1)); // DayOfWeek is invalid
```

Comment:
FWIW, I agree with that when the thing being repeated is minimal and less in size than an [InlineData], but here it's not.

This test is also missing [Fact] on it.